pkg/Cpanel/Security/Advisor/Assessors/Brute.pm

```diff
@@ -81,4 +132,31 @@ sub _check_for_brute_force_protection {
     return 1;
 }
 
+sub check_lfd_running {
+    if ( -e "/var/run/lfd.pid" ) {
+        my $fh_in;
+        if ( open( $fh_in, "<", "/var/run/lfd.pid" ) ) {
+            my $v_pid;
+            $v_pid = do { 
+                local $/; 
+                $v_pid = <$fh_in>;
+            };
+            close $fh_in;
+            chomp $v_pid;
+            if ( open( $fh_in, "<", ( "/proc/" . $v_pid . "/cmdline" ) ) ) {
```

Comment:
Suggest PsParser since it will normalize cmdline

$parsed_ps = Cpanel::PsParser::fast_parse_ps( 'want_pid' => $v_pid );
if ($parsed_ps && $parsed_ps->[0]->{'command'} =~ m{^lfd}) {
 return 1;
}